frontend/src/scenes/insights/LineGraph/LineGraph.tsx

```diff
@@ -60,6 +60,7 @@ interface LineGraphProps {
     tooltip?: TooltipConfig
     isCompare?: boolean
     incompletenessOffsetFromEnd?: number // Number of data points at end of dataset to replace with a dotted line. Only used in line graphs.
+    aggregationGroupTypeIndex?: number | null
```

Comment:
Using two bottom values is kind of sus and should likely get refactored to a better union, e.g. 

```
labelGroupType: number | 'people' | 'none'
```

Seems like a more consistent solution. That said, ymmv